### DISCUSSION TITLE
0.3.2

### DISCUSSION THREAD

**Discussion by schickling** [contentlayerdev/contentlayer#433] [Announcements]
ℹ️ **[TLDR]** Bug fixes for `next dev`, Support for `next export`, bug fixes and updated dependencies

## Improved `next-contentlayer` integration

As part of `0.3.2` we've overhauled the `next-contentlayer` integration with the goal of making it more stable and less dependent on implementation details of Next.js. This fixes #415 and #416 (thanks @kamto7).

As part of this effort (and by no longer relying on the `redirects` workaround) Contentlayer now also works with `next export`. (closes #426)

## Other Improvements

- Fix: Opentelemetry version incompatibility with next 13.2 (closes #407 - thanks @jgillich)
- Fix: Type resolution when using modern TypeScript module resolution (closes #373 - thanks @jrolfs)
- Fix: Korean file names are not supported (closes #431 - thanks @mi-reu)
- Fix: `contentDirInclude` didn't work in some cases (closes #383 - thanks to @teobler)

## Note about state of the project

Please also take a look at #429 to read about the current state of the project. 💜

<hr /><em>This discussion was created from the release <a href='https://github.com/contentlayerdev/contentlayer/releases/tag/v0.3.2'>0.3.2</a>.</em>

**Comment by pedro757** [comment]
Can't see this version yet on npm

**Reply by DrBarnabus** [reply]
It's showing on npm now
